var/spack/repos/builtin/packages/python/python-2.7.17+-distutils-C++-fixup.patch

```diff
@@ -0,0 +1,120 @@
+This patch fixes a regression intoduced by python-2.7.17+-distutils-C++.patch
+and updates the distutils testsuite to pass with 2.7.16:2.7.18:
```

Comment:
I've never seen a comment like this in a patch file before; thanks for documenting this fix so thoroughly!

Reply:
It was just a cut and past from the comment above - where it looks a bit better and was nicer to edit.

But yes, I understand that it is good to have the documentation also right there in the patch.
Applying past experience of mine, and as I already wrote it for the review itself.

Adding it to the patch for the benefit of others (or one self later) without having to wade thru the pull request makes it easy to see what a few lines of patch do.